Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoNetworks.yaml

```diff
@@ -17,25 +17,25 @@ References:
 - Title: Palo Alto Common Event Format Integration Guide [pdf]
   Link: https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/cef/pan-os-10-0-cef-configuration-guide.pdf
 Description: |
-  This ASIM parser supports normalizing the Palo Alto Networks CEF log to the ASIM WebSession normalized schema.
-ParserName: ASimWebSessionPaloAltoNetworks
-EquivalentBuiltInParser: _ASim_WebSession_PaloAltoNetworks
+  This ASIM parser supports normalizing Palo Alto PanOS threat event logs delivered using CEF to the ASIM WebSession normalized schema. The Palo Alto threat events are generated by the Palo Alto URL filtering module and collected using the Palo Alto Networks connectors, Common Event Format (CEF) via AMA connector, or the Common Event Format (CEF) via Legacy Agent connector to the CommonSecurityLog table.
+ParserName: ASimWebSessionPaloAltoCEF
+EquivalentBuiltInParser: _ASim_WebSession_PaloAltoCEF
 ParserParams:
   - Name: disabled
     Type: bool
     Default: false
 ParserQuery: |
   let parser=(disabled:bool=false){
-      let EventLookup=datatable(DeviceAction:string,EventResult:string,HttpStatusCode:string)
+      let EventLookup=datatable(DeviceAction:string, DvcAction:string,EventResult:string,HttpStatusCode:string)
       [
-          "alert", "Success","200"
-          , "allow", "Success","200"
-          , "continue", "Success","200"
-          , "override", "Success","200"
-          , "block-continue", "Partial","200"
-          , "block-url", "Failure","503"
-          , "block-override", "Failure","302"
-          , "override-lockout", "Failure","503"
+          "alert", "Allow", "Success","200"
```

Comment:
Are you sure we will not get the network level actions (drop, reset* etc?). Easiest is just to support the entire palo alto list

Reply:
Done